### PR TITLE
Develop migration

### PR DESCRIPTION
**Pull request by gaborod16**
Last changes incoming from internal repository:

- Proper type definition for the Parameters (within contexts)
- Clarify `IntentView` differences
- Improve reporting of the Q&A classification
- README.md slight clarification regarding the Google API included

### CODE REVIEW

File: ondewo/nlu/context.proto [hunk 1]
```diff
@@ -108,6 +107,23 @@ service Contexts {
 
 // Represents a context.
 message Context {
+
```
Comment (FergusFettes) [inline ondewo/nlu/context.proto:110]: i spent ages trying to figure out what was wrong with my mini grpc client the other day till filip told me about this :'(

such is the risk I take always using develop I suppose

Syntax does seem clearer, and you are also getting rid of a google dependency?

File: ondewo/qa/qa.proto [hunk 2]
```diff
@@ -63,8 +63,8 @@ message GetAnswerRequest {
     // Threshold (minimal score) to give back retriever result
     float threshold_retriever = 5;
 
-    // Formula for overall score
-    string custom_score_formula = 6;
+    // Threshold (minimal score) overall probability
+    float threshold_overall = 6;
```
Comment (FergusFettes) [inline ondewo/qa/qa.proto:67]: good, more clear

**Review by FergusFettes — APPROVED** [review]
thanks for the heads up, I guess this review was more for my sake than yours ;)

File: ondewo/nlu/context.proto (hunk 1 above)
Comment (gaborod16) [reply ondewo/nlu/context.proto:110]: You mean the Struct dependency?
Yes, that won't be necessary anymore. Not in this file :)

**Pull request merged**